src/features/router/index.css

```diff
@@ -1,5 +1,9 @@
-.header-wrapper {
-    display: flex;
-    justify-content: space-between;
-    margin-bottom: 40px;
+.container {
+    padding: 20px;
+}
+
+@media (min-width: 768px) {
+    .container {
+        padding: 40px;
+    }
 }
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Nesting_at-rules#nesting_media_at-rule

Reply:
If you want to use like this:

.container {
    padding: 20px;
    @media (min-width: 768px) {
        .container {
            padding: 40px;
        }
    }
}

it will not work as we do not use SCSS

Reply:
1. I recommend you use `\`` symbols to have your code formatted.  E.i. with  3 symbols: 
 ```
.container {
  padding: 20px;
  media (min-width: 768px) {
    .container {
      padding: 40px;
    }
  }
}
 ```

2. I sent you a ref for MDN, not SASS documentation

Reply:
1) Ok, I will use ``` symbols
2) It doesn't work. You cant try by yourself